ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -623,6 +628,49 @@ def osd_encryption_verification():
             )
             raise ValueError("OSD is not encrypted")
 
+    if ocs_version > version.VERSION_4_6:
+        log.info("Verify luks header label for encrypted devices")
+        worker_nodes = get_worker_nodes()
```

Comment:
@PrasadDesala yeah im only checking osd's running on worker nodes. check line 633
`worker_nodes = get_worker_nodes()`

Reply:
```get_worker_nodes()``` returns all the OCP worker nodes. Suppose if the cluster has an additional worker node without any running OSD this check will fail as there wont be any encrypted device on it. So, we have to check the encryption only on the osd running worker nodes.
You can use the existing function https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/node.py#L621 to get the osd running nodes.

Reply:
Okay I understand.. as per the logic it will only fail if the osd doesn't have header info not if the node doesn't have an osd. But if there is already existing function which only gives me osd running nodes then its great